_posts/2020-01-30-announcing-kadalu-0.5.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Announcing Kadalu Kubernetes Storage 0.5"
+author: kadalu
+layout: post
+description: "The Kadalu team is happy to announce a new version of Kadalu Kubernetes Storage, 0.5. Kadalu Kubernetes Storage is a lightweight solution to provide storage for applications running in Kubernetes."
```

Comment:
We are excited to make available a new release of Kadalu Storage. The Kadalu Storage 0.5 continues to provide a lightweight solution to provision persistent storage for applications running in Kubernetes

Reply:
done